ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md

```diff
@@ -33,6 +33,13 @@ The `dockerhost.yml` playbook can deploy single, multiple and duplicate containe
 
 will deploy 1 Ubuntu 22.04 container, 1 Alpine 3.19 container and 3 Ubuntu 18.04 containers.
 
+If you would like to build an arm32 container on an arm64 dockerhost, pass the `build_arm` variable:
+
+```
+ansible-playbook -u root -i <host-file> AdoptOpenJDK_Unix_Playbook
+/dockernode.yml -t "deploy" -e "docker_images=u2204 build_arm=yes"
```

Comment:
Would it be better to say arm32?

Reply:
Yep I agree - arm is too ambiguous to use here IMHO

Reply:
Done